README.md

```diff
@@ -1 +1,5 @@
 # cruise-ships
+Javascript, Jest and Node required
```

Comment:
So, readme...

What you want in here is some important info like:
- installation instructions (ie cloning the repo and what commands are available),
- what technologies are required to use it, what the application dependencies are (`JavaScript`, `Jest` and `Node` in this case)
- a description of what the app actually does and what its features are,
- what you would do if you had more time (if applicable)
- your name

This [former students repo](https://github.com/StuartCowley/agnes-beviz-cruise-ships) for reference, has quite a detailed readme and is formatted nicely, something along these lines would make a big difference